test/boost/suite.yaml

```diff
@@ -1,4 +1,6 @@
 type: boost
+extra_scylla_cmdline_options:
+    - '--reactor-backend linux-aio'
```

Comment:
Shouldn't we just make this default for all tests instead of select test suites?